packages/client/tests/functional/interactive-transactions/__snapshots__/tests.ts.snap

```diff
@@ -11,48 +11,48 @@ Invalid \`prisma.$executeRaw()\` invocation:
 exports[`interactive-transactions (cockroachdb) () batching rollback 1`] = `
 
 Invalid \`prisma.user.create()\` invocation in
-/client/tests/functional/interactive-transactions/tests.ts:234:19
+/client/tests/functional/interactive-transactions/tests.ts:237:19
 
-  231  */
-  232 testIf(getClientEngineType() === ClientEngineType.Library)('batching rollback', async () => {
-  233   const result = prisma.$transaction([
-→ 234     prisma.user.create(
+  234  */
+  235 testIf(getClientEngineType() === ClientEngineType.Library)('batching rollback', async () => {
+  236   const result = prisma.$transaction([
+→ 237     prisma.user.create(
```

Comment:
Snapshots like this one, with paths to the source code and code lines should be sanitized, as they're subject to frequent changes.